docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -0,0 +1,122 @@
+
+# Micro-packaging
+
+Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+
+## Package a modular pipeline
+
+You can package a modular pipeline by executing: `kedro pipeline package <pipeline_name>`
+
+* This will generate a new [wheel file](https://pythonwheels.com/) for this pipeline.
+* By default, the wheel file will be saved into `src/dist` directory inside your project.
+* You can customise the target with the `--destination` (`-d`) option.
+
+When you package your modular pipeline, Kedro will also automatically package files from 3 locations:
+
+```text
+├── conf
+│   └── base
+│       └── parameters
+│           └── {{pipeline_name*}}  <-- All parameter file(s)
+└── src
+    ├── my_project
+    │   ├── __init__.py
+    │   └── pipelines
+    │       └── {{pipeline_name}}    <-- Pipeline folder
+    └── tests
+        ├── __init__.py
+        └── pipelines
+            └── {{pipeline_name}}    <-- Pipeline tests
+```
+
+Kedro will also include any requirements found in `src/<python_package>/pipelines/<pipeline_name>/requirements.txt` in the modular pipeline wheel file. These requirements will later be taken into account when pulling a pipeline via `kedro pipeline pull`.
+
+```eval_rst
+.. note::  Kedro will not package the catalog config files even if those are present in ``conf/<env>/catalog/<pipeline_name>.yml``.
+```
+
+If you plan to publish your packaged modular pipeline to some Python package repository like [PyPI](https://pypi.org/), you need to make sure that your modular pipeline name doesn't clash with any of the existing packages in that repository. However, there is no need to rename any of your source files if that is the case. Simply alias your package with a new name by running `kedro pipeline package --alias <new_package_name> <pipeline_name>`.
+
+In addition to [PyPI](https://pypi.org/), you can also share the packaged wheel file directly, or via a cloud storage such as AWS S3.
+
+## Package multiple modular pipelines
+
+To package multiple modular pipelines in bulk, run `kedro pipeline package --all`. This will package all pipelines specified in the `tool.kedro.pipeline.package` manifest section of the project's `pyproject.toml` file:
+
+```toml
```

Comment:
@lorenabalan is this command still valid?

Reply:
The command is valid but the explanation is still too modular-pipeline focused. `first_package` and `second_package` are names of python packages in `src/<package_name>`, not necessarily modular pipeline folders. If they want modular pipeline folders, they would have to be specific, i.e. `pipelines.first_package`, `pipelines.first_package`.

Reply:
Actually I see you sort of addressed this in the `note` below, but I'm wondering if that's visible enough. Happy to approve as is, and if you have other ideas for rewording it then great, if not that's fine.